pwiz_aux/msrc/utility/vendor_api/Waters/WatersRawFile.hpp

```diff
@@ -192,6 +220,8 @@ struct PWIZ_API_DECL RawData
                 hasProfile_ = hasProfile_ || Info.IsContinuum(itr.first);
         }
 
+        ReadAnalogChromatograms();

```

Comment:
If this function is always and only called by the constructor, make it a private function `readAnalogChromatograms`. Do you know what kind of performance impact this has? If it's more than a small fraction of a second, this initialization needs to be done on-demand so client code that doesn't care about analog signals, like Skyline, can open a file and get a single spectrum as instantly as possible.

Reply:
okay.

Reply:
I forgot about this point: I wouldn't expect the performance for this method to be extremely poor, but I can measure that.